fbw-a380x/src/systems/extras-host/index.ts

```diff
@@ -88,8 +111,15 @@ class ExtrasHost extends BaseInstrument {
     this.versionCheck = new VersionCheck(process.env.AIRCRAFT_PROJECT_PREFIX, this.bus);
     this.aircraftSync = new AircraftSync(process.env.AIRCRAFT_PROJECT_PREFIX, this.bus);
 
-    this.backplane.addInstrument('Clock', this.clock);
+    this.backplane.addPublisher('SimvarPublisher', this.simVarPublisher);
+    this.backplane.addPublisher('MsfsElectricsPublisher', this.msfsElectricsPublisher);
+    this.backplane.addPublisher('MsfsFlightModelPublisher', this.msfsFlightModelPublisher);
+    this.backplane.addPublisher('MsfsMiscPublisher', this.msfsMiscPublisher);
+    this.backplane.addPublisher('GroundSupportPublisher', this.groundSupportPublisher);
+
     this.backplane.addInstrument('PilotSeatManager', this.pilotSeatManager);
+    this.backplane.addInstrument('GPUManagement', this.gpuManagement);
```

Comment:
```suggestion
    this.backplane.addInstrument('GpuManagement', this.gpuManagement);
```

Minor correction for camel case (i.e. MSFS is also an abbreviation but MsfsElectricsPublisher is correct)